stdlib/kokorawr/kokorawr/functions/services/rps.js

```diff
@@ -12,47 +8,74 @@ let options = ['rock', 'paper', 'scissors']
 * @returns {string}
 */
 module.exports = (user='Bob', team=1, option='rock', context, callback) => {
-
-  let index = options.indexOf(option.toLowerCase());
-  let color = 'Red';
-  let otherColor = 'Blue';
-  if(team != 0) {
-    color = 'Blue';
-    otherColor = 'Red';
-  }
-
-  if(index == -1) {
-    return callback(null, `<@${user}>:${color}, ${option} is not a valid option`);
-  } else if(waitingTeam == team || waitingTeam == -1) {
-    // enque the user
-    waiting.push([user, index]);
-    waitingTeam = team;
-    return callback(null, `<@${user}>:${color} is waiting for a match...`);
-  } else {
-    // deque and do the result
-    let other = waiting.shift();
-    let otherTeam = waitingTeam+0;
-    if(waiting.length == 0) {
-      waitingTeam = -1;
+  lib.utils.storage.get('rps', (err, rps) => {
+    console.log(rps)
+    if (err) {
+      return callback(null, 'An error has occurred with your command.');
+    }
+    if (rps == null) {
+      rps = {};
```

Comment:
Now that I think about it, maybe having things like rps, c4, etc. as the names of the stored json are kinda bad. Maybe instead we should just be consistent and call them all stored or storedValue or storedJson or something...

Reply:
Should I bring back tracker then?

Reply:
that's not what I was talking about. I'm talking about doing
```stored = {};```
instead of 
```rps = {};```
and 
```bs = {};```
etc...

Reply:
whoops, fixed now :)